targets/TARGET_NUVOTON/TARGET_NUC472/trng_api.c

```diff
@@ -95,6 +96,13 @@ int trng_get_bytes(trng_t *obj, uint8_t *output, size_t length, size_t *output_l
             *output_length += 32;
             output += 32;
         }
+		
+		if (*output_length != length) {
+			trng_get(tmpBuff) ;
```

Comment:
tab or 8 spaces used for this block of code? we use 4 spaces. line 85 seems also out of alignment